### DISCUSSION TITLE
Is there any downside to enabling the extra mouse buttons?

### DISCUSSION THREAD

**Discussion by zahti** [runelite/runelite#16279] [Q&A]
By default, the extra mouse buttons (Mouse4, Mouse5) are blocked by default in the RuneLite settings. Is there any downside to enabling them?
![image](https://user-images.githubusercontent.com/38710634/219119360-6451d6cc-ea85-4267-8db5-3a3e767a0f60.png)

EDIT - Just found this thread: https://github.com/runelite/runelite/discussions/16122
What is the benefit of enabling them if we are not able to bind them to specific actions? Just would like to understand.

**Comment by Alexsuperfly** [comment]
the jagex client interprets them as left clicks
